packages/monorepo-builder/packages-tests/Merge/ComposerJsonMerger/Fixture/with_provide.json

```diff
@@ -0,0 +1,22 @@
+{
+  "provide": {
+    "psr/log-implementation": "1.0",
+    "php-http/async-client-implementation": "1.0",
+    "my/some-implementation": "1.0"
```

Comment:
Could you update one of version to include the separator `1.0|2.0` and the 2nd file `2.0|3.0` too, so we can test the merging and unique verison clearing?